packages/dendron-next-server/pages/vscode/seed-browser.tsx

```diff
@@ -0,0 +1,137 @@
+import {
+  SeedConfig,
+  SeedRegistryDict,
+  SEED_REGISTRY,
+} from "@dendronhq/common-all";
+import { Avatar, Button, Card, Layout, List, PageHeader } from "antd";
+import _ from "lodash";
+import React from "react";
+import { useThemeSwitcher } from "react-css-theme-switcher";
+import { AddToWorkspaceButton, GoToSiteButton } from "../../components/seeds";
+import { DendronProps, WorkspaceProps } from "../../lib/types";
+import seedStyles from "../../styles/scss/seeds.module.scss";
+
+export default function SeedBrowser({
+  ide,
+  seedRegistryData,
+  browser,
+}: DendronProps & { seedRegistryData: SeedRegistryDict } & WorkspaceProps) {
+  const { switcher, themes, currentTheme, status } = useThemeSwitcher();
+  const [_isDarkMode, setIsDarkMode] = React.useState(false);
+
+  if (status === "loading") {
+    return <div>Loading styles...</div>;
+  }
+
+  const seedDataToRender = _.values(seedRegistryData).map((data) => {
+    if (data) {
+      let payload: SeedConfig & { seedInWorkspace: boolean } = {
+        ...data,
+        seedInWorkspace: false,
+      };
+      if (_.includes(ide.seedsInWorkspace, data?.id)) {
+        payload = {
+          ...payload,
+          seedInWorkspace: true,
+        };
+      }
+
+      return payload;
+    }
+  });
+
+  const toggleDarkMode = () => {
+    setIsDarkMode((previous) => {
+      switcher({ theme: previous ? themes.light : themes.dark });
+      return !previous;
+    });
+  };
+
+  const { Meta } = Card;
+  const { Footer, Content } = Layout;
+
+  return (
+    <>
+      <Layout className={seedStyles.layout}>
+        <Content style={{ padding: "0 50px" }}>
+          <div className={seedStyles.contentDiv}>
+            <PageHeader
+              className={seedStyles.siteHeader}
+              title="Dendron Seed Registry"
+              subTitle="Add Knowledge Bases to your Workspace"
```

Comment:
change the subtitle here to something shorter/more catchy. eg. "A Better way to Know"

Reply:
I think that might be good for a browser version of this view, but for the VS Code version I think something a little more informative of what the seeds are for would be better for users? My intention here was to give a hint as to what clicking on 'Add Seed' button would do.